XamlControlsGallery/App.xaml

```diff
@@ -122,6 +122,7 @@
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
             </Style>
 
+            <x:Double x:Key="TeachingTipMinWidth">100</x:Double>
```

Comment:
We need this since we need to override the resource on a level that is above the flyout layer which only is the App.xaml since the text for the copy code button is too short and we would have a gap.